mathics/doc/tex/Makefile

```diff
@@ -1,10 +1,11 @@
-TEX = pdflatex
+#TEX = pdflatex
```

Comment:
We should probably remove this comment. It may  cause confusion in the future.

Reply:
This is the only issue I found in this commit.

Reply:
I have removed it. Shall I make a rebase first?

Reply:
> It's not necessary. There are no conflicts in your branch 😁️